src/index.html

```diff
@@ -2,11 +2,379 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport" content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <nav class="nav">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="moyo_logo"
+            class="nav__moyo-logo"
+          >
+        </a>
```

Comment:
logo is not part of the navigantion
header consists of 2 elements - logo and navigation
remove it from nav and I think test will work

Reply:
also give a and img tag classes according to BEM